index.rst

```diff
@@ -131,8 +131,8 @@ special characters for wildcard matching:
 The backslash character (``\\``) can be used to escape a character so it is
 not interpreted as a special character.
 
-The maximum length of a section name is 4096 characters. All sections
-exceeding this limit are ignored.
+Cores must accept section names with length up to 1024 characters.
```

Comment:
Request: "up to and including 1024".  That reduces the chance of an off-by-one error.

Reply:
good catch! Fixed. Thanks. .)